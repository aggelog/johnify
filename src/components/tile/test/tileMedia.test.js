import React from 'react';
import TileMedia from '../tileMedia';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<TileMedia url="www.google.com" showGif={true} previewUrl={"www.google.com"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});