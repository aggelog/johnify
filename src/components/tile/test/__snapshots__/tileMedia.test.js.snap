// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="tile-media__container"
>
  <div
    className="tile-media__item-inside"
  >
    <div
      className="tile-media__loading-container"
    >
      <div>
        <span
          className="l-1"
        />
        <span
          className="l-2"
        />
        <span
          className="l-3"
        />
        <span
          className="l-4"
        />
        <span
          className="l-5"
        />
        <span
          className="l-6"
        />
      </div>
    </div>
    <img
      alt="..."
      className="tile-media__image tile-media__image-gif-loading"
      onLoad={[Function]}
      src="www.google.com"
    />
  </div>
</div>
`;
